pyfolio/tears.py

```diff
@@ -503,10 +530,22 @@ def create_txn_tear_sheet(returns, positions, transactions,
         Will plot additional swippage sweep analysis.
          - See pyfolio.plotting.plot_swippage_sleep and
            pyfolio.plotting.plot_slippage_sensitivity
+    estimate_intraday: boolean, optional
+        Approximate returns for intraday strategies.
+        See description in create_full_tear_sheet.
     return_fig : boolean, optional
         If True, returns the figure that was plotted on.
     """
 
+    if (estimate_intraday is None and
```

Comment:
If we have more or less identical code 3 times should refactor that into a function too.